src/backend_model_instance.cc

```diff
@@ -631,6 +631,11 @@ TritonModelInstance::WarmUp()
         request->CaptureQueueStartNs();
         triton_requests.push_back(
             reinterpret_cast<TRITONBACKEND_Request*>(request.get()));
+
+        // For warmup requests we need to manually set ResponseFactory
+        // since they modify the callback after PrepareForInference has
+        // been called.
+        request->SetResponseFactory();
```

Comment:
consider moving above the requests.push_back() for consistency with other request setup calls.

Reply:
moved.